Profile new feature

DARSHcverse opened the pull request:
This is an updated version for the profile photo selection. The older version was simple just two buttons and ordinary but the updated version has a modern style with a taking photo, uploading from gallery and cropping image for the profile.

DARSHcverse committed:
Profile Image XML Design

Profile Image
XML Design is made, next is to firebase.

DARSHcverse committed:
Profile Image XML Design

Profile Image
XML Design is made, next is to firebase.

DARSHcverse committed:
Update UploadPhoto.java

DARSHcverse committed:
Updated version of uploading profile images

XinyaoCheng reviewed (commented):
Hi, @DARSHcverse , do you have any test video?

DARSHcverse committed:
Small Changes to Crop

I Updated the crop, so that when users select images they will be sent directly to crop.

DARSHcverse commented:
> Hi, @DARSHcverse , do you have any test video?

https://youtu.be/5YVaDY0Zq98
